base/dates/periods.jl

```diff
@@ -89,7 +89,7 @@ end
 for (op, Ty, Tz) in ((:*, Real, :P),
                    (:/, :P, Float64), (:/, Real, :P))
     @eval begin
-        function ($op){P<:Period}(X::StridedArray{P}, y::$Ty)
+        function ($op)(X::StridedArray{P}, y::$Ty) where P<:Period
```

Comment:
Unless I missed use of `P` in the method body, perhaps could simplify the signature to `($op)(X::StridedArray{<:Period}, y::$Ty)`?